test/network_test.go

```diff
@@ -1,33 +1,20 @@
 package test
 
-func (s *CLITestSuite) TestNetworkDescribe() {
+func (s *CLITestSuite) TestNetwork() {
```

Comment:
What's the motivation for combining these? This test suite will take 3x longer to run while debugging now

Reply:
Again, that's something we're trying to move away from. I'll open a ticket to fix the pipeline tests!

Reply:
Sounds good. I will separate these test cases into different test suites.